bitnami/harbor/templates/trivy/trivy-cm-envvars.yaml

```
@@ -24,6 +24,9 @@ data:
   SCANNER_TRIVY_SEVERITY: {{ .Values.trivy.severity | quote }}
   SCANNER_TRIVY_IGNORE_UNFIXED: {{ .Values.trivy.ignoreUnfixed | quote }}
   SCANNER_TRIVY_SKIP_UPDATE: {{ .Values.trivy.skipUpdate | quote }}
+  SCANNER_TRIVY_SKIP_JAVA_DB_UPDATE: {{ .Values.trivy.skipJavaDbUpdate | quote }}
+  SCANNER_TRIVY_DB_REPOSITORY: {{ .Values.trivy.dbRepository | quote }}
+  SCANNER_TRIVY_JAVA_DB_REPOSITORY: {{ .Values.trivy.javaDbRepository | quote }}
```

Only set these env. vars if value is provided:
```suggestion
  {{- if .Values.trivy.dbRepository }}
  SCANNER_TRIVY_DB_REPOSITORY: {{ .Values.trivy.dbRepository | quote }}
  {{- end }}
  {{- if .Values.trivy.javaDbRepository }}
  SCANNER_TRIVY_JAVA_DB_REPOSITORY: {{ .Values.trivy.javaDbRepository | quote }}
  {{- end }}
```